lms/djangoapps/django_comment_client/forum/tests.py

```diff
@@ -317,11 +317,11 @@ class SingleThreadQueryCountTestCase(ModuleStoreTestCase):
 
     @ddt.data(
         # old mongo with cache
-        (ModuleStoreEnum.Type.mongo, 1, 7, 5, 12, 7),
-        (ModuleStoreEnum.Type.mongo, 50, 7, 5, 12, 7),
+        (ModuleStoreEnum.Type.mongo, 1, 7, 5, 13, 7),
+        (ModuleStoreEnum.Type.mongo, 50, 7, 5, 13, 7),
```

Comment:
We should then change back the order of the calls.  As I wondered above, this may be a result of reordering the check for staff access.